src/sentry/static/sentry/app/components/activity/note/index.jsx

```diff
@@ -27,20 +30,47 @@ class Note extends React.Component {
     };
   }
 
-  onEdit = () => {
+  handleEdit = () => {
     this.setState({editing: true});
   };
 
-  onFinish = () => {
+  handleEditFinish = () => {
     this.setState({editing: false});
   };
 
-  onDelete = () => {
-    this.props.onDelete(this.props.item);
+  handleDelete = () => {
+    const {item, onDelete} = this.props;
+
+    if (!onDelete) {
+      return;
```

Comment:
Should this be an error? If someone triggered a delete and we have no handler something is probably broken.